src/data/actions/assets.js

```diff
@@ -21,3 +24,25 @@ export const filterUpdate = (filterKey, filterValue) => ({
   type: FILTER_UPDATED,
   data: { [filterKey]: filterValue },
 });
+
+export const deleteAssetSuccess = (assetId) => ({
+  type: DELETE_ASSET_SUCCESS,
+  assetId,
+});
+
+export const assetXHRFailure = (text) => ({
+  type: ASSET_XHR_FAILURE,
+  text,
+});
+
+export const deleteAsset = (assetsParameters, assetId) =>
+  dispatch =>
+    clientApi.deleteAsset(assetsParameters.courseId, assetId)
```

Comment:
I hesitate a little to have this function be called deleteAsset and the client API function to be deleteAsset, since they do semantically different things: one is an action creator and the other is actually making an API request. I'd like for the names to be differently, but I can't think of a not-clunky solution. deleteAsset here and requestAssetDelete in the API? Any thoughts? It's not a big deal, but I figure I'd rather resolve it now rather than later.

Reply:
What about `deleteAssetApi`? Since it's an API call? We can talk through this some more if you'd like too.